README.md

```diff
@@ -1,5 +1,6 @@
 [![Travis Build Status](https://api.travis-ci.org/xspec/xspec.svg?branch=master "Travis Build Status")](https://travis-ci.org/xspec/xspec)
 [![AppVeyor Build status](https://ci.appveyor.com/api/projects/status/github/xspec/xspec?branch=master&svg=true "AppVeyor Build Status")](https://ci.appveyor.com/project/xspec/xspec/branch/master)
+[![Azure Pipelines Build Status](https://dev.azure.com/xspec/xspec/_apis/build/status/xspec.xspec?branchName=azure_pipelines_test)](https://dev.azure.com/xspec/xspec/_build/latest?definitionId=1&branchName=azure_pipelines_test)
```

Comment:
This URL scheme and hier-part look fine, but `branchName=azure_pipelines_test` looks suspicious. `=master`, perhaps?

Reply:
Thanks for spotting this, I used the branch of this pull request for showing the badge and the link but forgot to change it to `master` before initiating this pull request. This is now done in 
990c307. Let me know if you spot anything else.